docs/workload-development-kit/workload-environment.md

```diff
@@ -37,6 +37,31 @@ The workloads can run in two environments: local and cloud. In local (devmode),
 
 ## Development environment
 
+### Workload NuGet package structure
+
+The Workload is packaged as a NuGet package, combining backend (BE) and frontend (FE) components. The structure adheres to specific naming conventions and is enforced by Fabric for consistency across upload scenarios.
+The NuGet package designed to represent Workloads is structured to include both backend (BE) and frontend (FE) components.
+
+#### Backend (BE) Structure
+
+The BE segment comprises XML files that define the Workload and its associated Items, which are essential for registration within the Fabric system.
+
+##### Key Components:
+- **`WorkloadManifest.xml`**: The critical manifest file for your Workload, required to have this exact name for Fabric's verification.
+- **`Item1.xml`, `Item2.xml`, ...**: Manifests for individual Items with flexible naming, following the XML format.
+
+#### Frontend (FE) Structure
+
+The FE section contains JSON files detailing the Product and Items for the frontend, along with an 'assets' directory for icons.
```

Comment:
FE = Frontend; JSON = .json; Product = product; Items = items; why 'assets' not assets?

Reply:
'assets' is to implicitly indicate that there is and "assets" folder